### ISSUE TITLE
Broken paddings, yet again

### ISSUE THREAD

**Issue by ekuiter** [SoftVarE-Group/Course-on-Software-Product-Lines#16]
Due to commit 26dba4b (updated slide template), some paddings in boxes have now been "improved":

![grafik](https://github.com/SoftVarE-Group/Course-on-Software-Product-Lines/assets/4258713/563bd35f-20d5-407c-bd11-94ca52405fc3)

This is due to additional `\vspace` commands, which I originally added to fix some broken paddings. This is not an issue in the slide template. The correct fix is go to through the slides and remove some of these `\vspace` commands. I will do it when I find the time.

**Comment by Benno2000** [comment]
I just took a look at this issue. It does not seem to be an issue with the `fancycolumns` environment, as a math-environment inside a single colorbox also has the space above it.

However, this seems to be the normal behavior of a math display environment as it is usually used inside a text and not at the beginning of a new paragraph: https://tex.stackexchange.com/a/337107

To solve this, the paddings of the math display environments can be adjusted like shown in the post mentioned above. So for example you could use `\setlength{\abovedisplayskip}{0ex}` (or something < 0 if the padding is still to large) to set the top padding of the environments to 0 at the top of the document (after `\begin{document}`).

**Comment by ekuiter** [comment]
If I understand it correctly, this `\setlength` would affect all boxes, those with only text included. So it would be difficult to distinguish both kinds of boxes ... which probably means we have to stick with `\vspace`, I guess.

> Display-math environments should never, ever occur at the start of a paragraph.

To me, that seems to be an unreasonable expectation. But if it's the official way LaTeX handles this, there is probably litte we can do

**Comment by tthuem** [comment]
We should then at least use a common macro such that we can easily change that vspace by changing a single location.

**Comment by ekuiter** [comment]
Unfortunately, the space itself depends also on the content that follows (e.g., if it is `\small`). :/

**Comment by tthuem** [comment]
Isn't if feasible to insert some dummy content of height zero on top to avoid this extra handling? This could then be added using a macro.

**Comment by ekuiter** [comment]
Good idea, unfortunately it does not seem to work (I tried \hphantom), because I need to set a **negative** space, height zero does not cut it.

I also experimented with different units of length (millimeters are independent of the selected font, but this does not work either).

**Comment by tthuem** [comment]
I would prefer to have a minimal example and to discuss this further for the template. Florian may be able to help then.
